v-next/hardhat/src/internal/cli/helpers/utils.ts

```diff
@@ -1,7 +1,10 @@
 import type { ParameterType } from "@ignored/hardhat-vnext-core/config";
 import type { Task } from "@ignored/hardhat-vnext-core/types/tasks";
 
-export const GLOBAL_OPTIONS = [
+export const GLOBAL_OPTIONS: Array<{
```

Comment:
how about `Pick<GlobalOption, "name" | "description">`

Reply:
Let's cover this in #5418